src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -37,64 +38,59 @@ class CrudField {
             this.change(closure);
         }
 
-        hide(e) {
-            this.wrapper.hide();
-            this.input.trigger('backpack:field.hide');
+        show(value = true) {
+            this.wrapper.toggleClass('d-none', !value);
+
+            this.input.trigger(`backpack:field.${value ? 'show' : 'hide'}`);
             return this;
         }
 
-        show(e) {
-            this.wrapper.show();
-            this.input.trigger('backpack:field.show');
-            return this;
+        hide() {
+            return this.show(false);
         }
 
-        enable(e) {
-            this.input.removeAttr('disabled');
-            this.input.trigger('backpack:field.enable');
+        enable(value = true) {
+            this.input.attr('disabled', !value && 'disabled');
+
+            this.input.trigger(`backpack:field.${value ? 'enable' : 'disable'}`);
             return this;
         }
 
-        disable(e) {
-            this.input.attr('disabled', 'disabled');
-            this.input.trigger('backpack:field.disable');
-            return this;
+        disable() {
+            return this.enable(false);
         }
 
-        require(e) {
-            this.wrapper.removeClass('required').addClass('required');
-            this.input.trigger('backpack:field.require');
+        require(value = true) {
+            this.wrapper.toggleClass('required', value);
+
+            this.input.trigger(`backpack:field.${value ? 'require' : 'unrequire'}`);
             return this;
         }
 
-        unrequire(e) {
-            this.wrapper.removeClass('required');
-            this.input.trigger('backpack:field.unrequire');
-            return this;
+        unrequire() {
+            return this.require(false);
         }
 
-        check(e) {
-            this.wrapper.find('input[type=checkbox]').prop('checked', true).trigger('change');
+        check(value = true) {
+            this.wrapper.find('input[type=checkbox]').prop('checked', value).trigger('change');
             return this;
         }
 
-        uncheck(e) {
-            this.wrapper.find('input[type=checkbox]').prop('checked', false).trigger('change');
-            return this;
+        uncheck() {
+            return this.check(false);
         }
     }
 
     /**
      * Window functions that help the developer easily select one or more fields.
      */
     window.crud = {
-        field: function(fieldName) {
-            return new CrudField(fieldName);
-        },
-        fields: function(fieldNamesArray) {
-            return fieldNamesArray.map(function(fieldName) {
-                return new CrudField(fieldName);
-            });
-        }
-    }
-</script>
+        ...window.crud,
```

Comment:
This is a fail safe, if someone is using `window.crud` on it's side, our JS Field library will not **overwrite** it, but it **override** it.

Even we do use this variable `window.crud` on List operation, so I think it's safer this way.